### PR TITLE
docs: Add a section on migrating from an existing configuration

### PR DESCRIPTION
**Pull request by wakamsha**
## Summary

Write a section on migration from an existing configuration in `README.md`.

## Others

- The plugin name for Next.js was incorrect, so it has been fixed.
- `eslint-plugin-jest-dom` was missing from the rule sets and dependent packages correspondence table, so I added it.

## References

- #263

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ We also provide various other rule sets that you can configure to suit your proj
...
 |      `jsdoc` | Contains JSDoc recommended rules                      | [`eslint-plugin-jsdoc`](https://github.com/gajus/eslint-plugin-jsdoc)                                                                                                                                                                                                               |
-|       `next` | Contains Next.js recommended rules                    | [`eslint-plugin-next`](https://github.com/vercel/next.js/tree/canary/packages/eslint-plugin-next)                                                                                                                                                                                   |
+|       `next` | Contains Next.js recommended rules                    | [`@next/eslint-plugin-next`](https://github.com/vercel/next.js/tree/canary/packages/eslint-plugin-next)                                                                                                                                                                             |
```
Comment (wakamsha) [inline README.md:60]: The plugin name for Next.js was incorrect, so it has been fixed.

File: README.md [hunk 2]
```diff
@@ -57,11 +57,11 @@ We also provide various other rule sets that you can configure to suit your proj
...
 |  `storybook` | Contains Storybook recommended rules                  | [`eslint-plugin-storybook`](https://github.com/storybookjs/eslint-plugin-storybook)                                                                                                                                                                                                 |
-| `test/react` | Contains Jest and React Testing Library rules         | [`eslint-plugin-jest`](https://github.com/jest-community/eslint-plugin-jest) <br> [`eslint-plugin-testing-library`](https://github.com/testing-library/eslint-plugin-testing-library)                                                                                               |
+| `test/react` | Contains Jest and React Testing Library rules         | [`eslint-plugin-jest`](https://github.com/jest-community/eslint-plugin-jest) <br> [`eslint-plugin-jest-dom`](https://github.com/testing-library/eslint-plugin-jest-dom) <br> [`eslint-plugin-testing-library`](https://github.com/testing-library/eslint-plugin-testing-library)    |
```
Comment (wakamsha) [inline README.md:64]: `eslint-plugin-jest-dom` was missing from the rule sets and dependent packages correspondence table, so I added it.

**Review by taigakiyokawa — APPROVED** [review]
LGTM 🙆‍♂️ (for now. This PR looks like a draft so please re-request review to me if you add more changes other than recovering CI failed.)

**Comment by wakamsha** [comment]
@taigakiyokawa 
Sorry, I forget to clear the reviewer list.
This is still in Draft, so I'll ask for another review once it's officially open 🙏

**Pull request merged**